central/scannerdefinitions/handler/handler.go

```diff
@@ -500,31 +500,43 @@ func (h *httpHandler) validateV4DefsVersion(zipPath string) error {
 	}
 	defer utils.IgnoreError(zipR.Close)
 
+	validate := func(zipF *zip.File) error {
+		// extract the Scanner V4 file out of the ZIP.
+		defs, err := h.openFromArchive(zipPath, zipF.Name)
+		if err != nil {
+			return errors.Wrap(err, "opening Scanner V4 definitions")
+		}
+		defer utils.IgnoreError(defs.Close)
+		// extract the manifest file out of the extracted Scanner V4 defs file.
+		mf, err := h.openFromArchive(defs.Name(), scannerV4ManifestFile)
+		if err != nil {
+			return errors.Wrap(err, "opening Scanner V4 definitions manifest")
+		}
+		offlineV, err := readV4ManifestVersion(mf)
+		defer utils.IgnoreError(mf.Close)
```

Comment:
Super nit: swap order
```suggestion
		defer utils.IgnoreError(mf.Close)
		offlineV, err := readV4ManifestVersion(mf)
```